### PR TITLE
New package: python3-weasyprint

### PR DESCRIPTION
**Pull request by grafov**
[Weasyprint](https://weasyprint.org/) used for example by Pandoc as a printing engine. It could be used as a standalone HTML convertor too.

Also the package add dependencies that missed in Void yet:
* python3-cairosvg
* python3-tinycss2
* python3-cssselect2
* python3-pyphen

### CODE REVIEW

**Comment by Hoshpak** [comment]
I havent revied the changes themselves yet but this PRs git history is already making it unfit for merge. Please revise the PR so that only the "New package:" commits are part of the branch, especially the merge commits need to go.

If you feel the need to pull in changes from master in the future while working on your branch, please use rebase instead of merge. It will pull the commits from master and then add your commits at the top, avoiding merge commits.

File: srcpkgs/python3-cairosvg/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'python3-cairosvg'
+pkgname=python3-cairosvg
```
Comment (Chocimier) [inline srcpkgs/python3-cairosvg/template]: I would use same case as upstream: CairoSVG

File: srcpkgs/python3-cairosvg/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+version=2.4.0
+revision=1
+archs=noarch
+wrksrc="CairoSVG-${version}"
+build_style=python3-module
+pycompile_module="cairosvg"
+hostmakedepends="python3-setuptools"
+depends="python3-cairocffi python3-Pillow python3-defusedxml"
```
Comment (Chocimier) [inline srcpkgs/python3-cairosvg/template]: also cairo

File: srcpkgs/python3-cssselect2/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+version=0.2.1
+revision=1
+archs=noarch
+wrksrc="cssselect2-${version}"
+build_style=python3-module
+pycompile_module="cssselect2"
+hostmakedepends="python3-setuptools"
+short_desc="CSS selectors for Python3 ElementTree"
+maintainer="Alexander I.Grafov <grafov@gmail.com>"
+license="3-clause-BSD"
```
Comment (Chocimier) [inline srcpkgs/python3-cssselect2/template]: SPDX names that BSD-3-Clause

File: srcpkgs/python3-pyphen/template [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-pyphen'
+pkgname=python3-pyphen
```
Comment (Chocimier) [inline srcpkgs/python3-pyphen/template]: python3-Pyphen

File: srcpkgs/python3-weasyprint/template [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-weasyprint'
+pkgname=python3-weasyprint
```
Comment (Chocimier) [inline srcpkgs/python3-weasyprint/template]: As this is mainly comandline tool, name should be WeasyPrint.

File: srcpkgs/python3-weasyprint/template [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+version=48
+revision=1
+archs=noarch
+wrksrc="WeasyPrint-${version}"
```
Comment (Chocimier) [inline srcpkgs/python3-weasyprint/template]: remove wrksrc= after rename

File: srcpkgs/python3-weasyprint/template [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+build_style=python3-module
+pycompile_module="weasyprint"
+hostmakedepends="python3-setuptools"
+depends="python3-cffi python3-html5lib python3-cssselect2 python3-pyphen python3-cairosvg python3-tinycss2"
```
Comment (Chocimier) [inline srcpkgs/python3-weasyprint/template]: also pango

Comment (Chocimier) [inline srcpkgs/python3-weasyprint/template]: and also python3-setuptools in depends

**Comment by travankor** [comment]
inactive?

**Comment by ericonr** [comment]
Seems to be the case

**Pull request closed without merging**